Reservia.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<!------------------------------------------------------------------- LINK CSS ----------------------------------------------------------->
+<html lang="fr">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+	<title>Reservia</title>
+	<link rel="stylesheet" href="./Reservia_files/css/style.css">
+	<link rel="stylesheet" href="./Reservia_files/css/header.css">
+	<link rel="stylesheet" href="./Reservia_files/css/navigation.css">
+	<link rel="stylesheet" href="./Reservia_files/css/lodging.css">
+	<link rel="stylesheet" href="./Reservia_files/css/activities.css">
+	<link rel="stylesheet" href="./Reservia_files/css/footer.css">
+</head>
+<!------------------------------------------------------------------- END LINK CSS ------------------------------------------------------->
+
+
+<body> 
+<!------------------------------------------------------------------- HEADER ------------------------------------------------------------->
+    <header class="pageHead">
+        <img class="pageHead__logo" src="./Reservia_files\Pictures/Header/reservia_logo.svg" alt="logo Reservia">
+        <nav class="pageHead__nav">
+            <a class="pageHead__navItem" href="hebergement.html">Hébergements</a>
+            <a class="pageHead__navItem" href="Activities.html">Activités</a>
+        </nav>
+        <a class="pageHead__itemRegister" href="page_register.html">S'inscrire</a>
+    </header>
+<!------------------------------------------------------------------END HEADER ------------------------------------------------------------>
+
+
+
+
+<!------------------------------------------------------------------- NAVIGATION ---------------------------------------------------------->
+    <h2 id="h2navigation">Trouvez votre hébergement pour des vacances de reves</h2>
```

Comment:
this h2 is not having the same style as the one on the other sections ?

Reply:
new indentation performed

Reply:
I just transformed the H2 into H1

Reply:
don't forget to push your commit because i don't see the changes yet.